### DISCUSSION TITLE
CSS / Large Icons on new install of LinkAcev1.9.1

### DISCUSSION THREAD

**Discussion by craigmckay** [Kovah/LinkAce#388] [Support] [answered]
Good evening!

LinkAce looks like exactly what I've been looking for, for years since delicious disappeared!

I've installed it at https://linkace.southesk.com/public/ and as you can see, something is wrong with the CSS or the responsive settings or something!

Can anyone help?

Many thanks,
Craig

**Comment by Kovah** [comment, marked as answer]
Hi, your web server must point to the `/public` directory, instead of the base install directory. Otherwise the app is not usable.

**Reply by craigmckay** [reply]
Fantastic Kovah!  Thanks so much - I updated the subdomain mapping and it's working perfectly.  Looks excellent.
